package/regenda/package

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(regenda)
+pkgdesc="Caldav agenda app"
+url=https://github.com/tenJirka/reGenda
+pkgver=0.0.1-1
+timestamp=2023-08-23T19:40Z
+section="utils"
+maintainer="tenJirka <jirka.steh@gmail.com>"
+license=GPL3
+installdepends=(python3 python3-pip python3-lxml)
+flags=(patch_rm2fb)
+
+source=(https://github.com/tenJirka/reGenda/archive/refs/tags/0.0.1.zip)
+sha256sums=(00d6a1e2f6b59f86ba47ea5c3444f24e112d24eb62082802653fe4649153da01)
+
+package() {
+    install -D -m 644 -t "$pkgdir"/opt/etc/draft/ "$srcdir"/regenda.draft
+    install -D -m 644 -t "$pkgdir"/opt/etc/reGenda/ "$srcdir"/config.yml.example
+    install -D -m 644 "$srcdir"/icon.png "$pkgdir"/opt/etc/draft/icons/reGenda.png
+    install -D -m 755 -t "$pkgdir"/opt/reGenda/ "$srcdir"/run.py
```

Comment:
While we do have a couple packages that put their files directly in `/opt/{pkgname}`, it would be better to place them further down the file structure. If possible, I would recommend moving files to places under `/opt` that mirror the [file-hierarchy](https://man.archlinux.org/man/file-hierarchy.7.en)

Reply:
I'm not sure how to do this. The configuration files are already in /opt/etc, I can't move the executable to /opt/bin because it has to be in the same directory as all the libraries. And now all pip modules must be there too.
If you give more clues as to how you'd like it, I'll look into it.

Reply:
`/opt/usr/lib/reGenda` would probably be a good location.

> /usr/lib/
Static, private vendor data that is compatible with all architectures (though not necessarily architecture-independent). Note that this includes internal executables or other binaries that are not regularly invoked from a shell. Such binaries may be for any architecture supported by the system. Do not place public libraries in this directory, use $libdir (see below), instead.

Reply:
Oh, I totally agree. 
Already moved here.